cli/auth_nkey_command.go

```diff
@@ -256,3 +272,106 @@ func (c *authNKCommand) verifyAction(_ *fisk.ParseContext) error {
 
 	return nil
 }
+func (c *authNKCommand) sealAction(_ *fisk.ParseContext) error {
+	var err error
```

Comment:
not needed its declared 2 lines below, kp probably also not needed to be pre-declared as you can change `=` to `:=` on line 285 with no ill side effects